Resources/config/security.xml

```diff
@@ -9,8 +9,7 @@
         </service>
         <service id="sonata.media.security.superadmin_strategy" class="Sonata\MediaBundle\Security\RolesDownloadStrategy">
             <argument type="service" id="translator"/>
-            <argument/>
-            <!-- Either security.context or security.authorization_checker -->
+            <argument type="service" id="security.authorization_checker"/>
```

Comment:
@OskarStark [this class was removed](https://github.com/sonata-project/SonataMediaBundle/pull/1165/files#diff-84cf3aa70e3d2b787e0810bf807f4406)